README.md

```diff
@@ -55,27 +55,25 @@ npm install -g @nanoapi.io/napi
 
 ```bash
 napi init
-napi ui
+napi split configure
+napi split
 ```
 
 This will initialize a new NanoAPI project and open the UI for inspecting and refactoring your codebase.
 
-## CLI Usage
+**Note:** `napi` works off a simple system of annotation. These annotation allow `napi` to be integrated into any existing projects. Annotating a program is needed before using the split functionalities see [Refactoring with Annotations](#refactoring-with-annotations).
 
-`napi` provides the following commands:
+## CLI Usage
 
-```bash
-init                  Initialize the NanoAPI project
-ui [entrypoint]       Open the UI to inspect and refactor the codebase
-split <entrypoint>    Transform the codebase into microservices
-napi <platform> annotate <entrypoint> [--token=<key>] Automatically generate annotations for the codebase via OpenAI
-```
+`napi --help` will provides an overview off all commands
```

Comment:
I just leave it like this. To much repetition. We have the doc for more detail. The help command provide the same info, and is always up to date.